scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableConfig.scala

```diff
@@ -1,4 +1,5 @@
-package scalafix.internal.config
+package scalafix
```

Comment:
I generally avoid this pattern since it can easily introduce cycles in your codebase. `import scalafix.config.CustomMessage` is fine in our own code. However, I'm not so consistent myself 😸